script.js

```diff
@@ -0,0 +1,113 @@
+function ReadElement(selector) {
+  return document.querySelector(selector);
+}
+
+const menu = ReadElement('#menu-image');
+const nav = ReadElement('.mobile-nav');
+const closeNav = ReadElement('.cross');
+const take = ReadElement('mobile-nav-ul');
+
+function addMenu(selected, modalPart, opener) {
+  return selected.addEventListener('click', () => {
+    modalPart.classList.add(opener);
+    modalPart.style.top = '0px';
+  });
+}
+
+function removeMenu(selected, modalPart, remover) {
+  return selected.addEventListener('click', () => modalPart.classList.remove(remover));
+}
```

Comment:
- [ ] You did great implmenting the mobile menu, :+1: :100: But looking the menu, you can see that the button is overlaping the menu item, I suggest you adjust the styling as this dose4nt look good in to the end user. :+1: :smile: 

- - Screenshot:
![Screenshot from 2023-05-25 12-00-48](https://github.com/Kgomotso196/FirstCapstoneProject/assets/107632780/3fdd5c67-b055-42c2-ab60-f5e508acbcf2)